app/assets/javascripts/components/notes/Note.js

```diff
@@ -1,49 +1,35 @@
 import React from 'react';
 
-class Note extends React.Component {
-  constructor(props) {
-    super(props);
-    this._handleDelete = this._handleDelete.bind(this);
-  }
+const Note = ({ note, disabled, handleDelete }) => {
 
-  _handleDelete() {
-    const { note, handleDelete } = this.props;
-    handleDelete(note);
+  const _handleDelete = () => {
+    handleDelete(note)
   }
 
-  parseNote(note) {
+  const parseNote = () => {
     return ({
       __html: window.md.makeHtml(note.escape('note'))
     });
   }
-
-  renderDelete() {
-    return (
-      <span>
-        - <span
-            onClick={this._handleDelete}
+  return (
+    <div className='note'>
+      <div dangerouslySetInnerHTML={parseNote(note)}></div>
+      <div className='note_meta'>
+        <span className='user'> { note.attributes.user_name } </span> -
+        <span className='created_at'> { note.attributes.created_at } </span>
+        {!disabled && <span>
+          - <span
```

Comment:
Since this is a clickable element, what do you think about turning this into a button? and then on the test we can fetch it by role instead of testid cc: @talyssonoc